syntax = "proto3";

import "node.proto";

service Client {
    rpc ClientLookupReply(LookupReply) returns (Empty) {}
    rpc ClientInsertReply(InsertReply) returns (Empty) {}
}